src/components/BookList.js

```diff
@@ -0,0 +1,32 @@
+const bookList = [
+  {
+    category: 'Action',
+    title: 'The Hunger Games',
+    author: 'Suzanne Collins',
+    chapter: 'Chapter 17',
+    percentage: '64%',
+  },
+  {
+    category: 'Science Fiction',
+    title: 'Dune',
+    author: 'Fran Herbert',
+    chapter: 'Chapter 3: A lesson Learned',
+    percentage: '8%',
+  },
+  {
+    category: 'Economy',
+    title: 'Capital in the Twenty-first Century',
+    author: 'Thomas Piketty',
+    chapter: 'CURRENT CHAPTER',
+    percentage: '0%',
+  },
+  {
+    category: 'Economy',
+    title: 'Capital in the Twenty-first Century',
+    author: 'Thomas Piketty',
+    chapter: 'CURRENT CHAPTER',
+    percentage: '80%',
+  },
+];
+
```

Comment:
- @efmomoh as requested by the previous two reviewers the milestone requires that rather than eporting the state as a defualt which you are currently doing, you are to render the `<Book>` component in the `<BookList>` component and pass in a book as a prop. see the code snippet below for guide and reference :+1: :sparkles: 
- code snippets
```javascript
return (
    <div>
      {bookList.map((book) => (
        <Book
          key={book.id}
          id={book.id}
          title={book.title}
          author={book.author}
          category={book.category}
        />
      ))}
    </div>
  );
```